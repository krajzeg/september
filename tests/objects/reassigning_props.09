a := 12
a := 42
print("This should not be a problem, and should print 42:", a)

b := [[fourtyTwo: 42]]
b::method = 1
b:::method = { this.fourtyTwo }
print("This should use a method to print 42:", b.method())

print("This should print Ann, Belle, Carol in order:")
for (person) in (["Ann", "Belle", "Carol"]) {
	name := person
	print(name)
}
