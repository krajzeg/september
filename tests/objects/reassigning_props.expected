This should not be a problem, and should print 42: 42
This should use a method to print 42: 42
This should print Ann, Belle, Carol in order:
Ann
Belle
Carol
